### PR TITLE
Foma

### PR DESCRIPTION
**Pull request by fabysshk**
component!!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,17 +3,24 @@ import './stylesheets/main.scss';
 import '@babel/polyfill';
 
 import {log, createEl, appendTo, text, css} from "./js/helpers";
-import './js/functions'; // Empty
+import {} from './js/functions'; // Empty
```
Comment (fabysshk) [inline src/index.js]: @DZakh

File: src/index.js [hunk 2]
```diff
@@ -3,17 +3,24 @@ import './stylesheets/main.scss';
 import '@babel/polyfill';
 
 import {log, createEl, appendTo, text, css} from "./js/helpers";
```
Comment (fabysshk) [inline src/index.js]: not used log->remove this

File: src/index.js [hunk 3]
```diff
@@ -3,17 +3,24 @@ import './stylesheets/main.scss';
...
-    color: 'red',
-    background: 'blue'
-});
-appendTo(document.body, div);
+import {header} from "./js/components/header";
+
+const links = [
+    {
+        path: "/",
+        name: "home"
```
Comment (fabysshk) [inline src/index.js]: 'wtf' dblquotes

File: src/index.js [hunk 4]
```diff
@@ -3,17 +3,24 @@ import './stylesheets/main.scss';
...
+    },
+    {
+        path: "/about",
+        name: "about as"
```
Comment (fabysshk) [inline src/index.js]: some

**Comment by DZakh** [comment]
Wait, I don't think we should make a pull request into master branch!

**Comment by DZakh** [comment]
> component!!

In the project we don't need components.

**Pull request closed without merging**